polyfill/test/resolve.cookbook.mjs

```diff
@@ -0,0 +1,25 @@
+/*
+ ** Copyright (C) 2018-2019 Bloomberg LP. All rights reserved.
+ ** This code is governed by the license found in the LICENSE file.
+ */
+
+import fs from 'fs';
+const PKG = JSON.parse(fs.readFileSync('package.json', { encoding: 'utf-8' }));
+export function resolve(specifier, parent, defaultResolve) {
+  if (specifier === PKG.name) {
+    specifier = new URL('../lib/temporal.mjs', import.meta.url).toString();
+  }
+  return defaultResolve(specifier, parent);
+}
+
+export async function transformSource(
```

Comment:
This wasn't working for me until I figured out that this hook was only added in Node 12.16.0. So we should require that version somewhere. It probably needs to be both in package.json and in the CI configuration.

Reply:
be sure to add it to the "engines" field, which is advisory but usually sufficient